identity/ideth/factory_contract.go

```diff
@@ -137,7 +137,7 @@ func bindFactoryContract(address common.Address, caller bind.ContractCaller, tra
 // sets the output to result. The result type might be a single field for simple
 // returns, a slice of interfaces for anonymous returns and a struct for named
 // returns.
-func (_FactoryContract *FactoryContractRaw) Call(opts *bind.CallOpts, result interface{}, method string, params ...interface{}) error {
+func (_FactoryContract *FactoryContractRaw) Call(opts *bind.CallOpts, result *[]interface{}, method string, params ...interface{}) error {
```

Comment:
Why did it change? Are we using a newer version of `abigen`?

Reply:
yes